### ISSUE TITLE
Invasive Nicotiana glauca is not found in central Saudi Arabia only in the southwestern part of the country.

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#3877]
**Invasive Nicotiana glauca is not found in central Saudi Arabia only in the southwestern part of the country.**

Invasive Nicotiana glauca is not found in central Saudi Arabia only in the southwestern part of the country. I am sure of this information because I have done several studies on this type and have published two research papers on this invasive species. In addition, my Ph.D. studies were all about the vegetation that was invaded by Nicotiana glauca in my country, Saudi Arabia.

-----

User: [See in registry](https://www.gbif.org/api/feedback/user/0d3813f516b0f41287753a3bb20b1b4c:15ffe4f60efa7e5010ad8aefaebd73fa61fae0897a9cde1f0c30407fae7648dfd6c6de79ff3251e67d3e5a17848c874ad8707e7aac85c4b178640dd7d3771af4)
System: Chrome 97.0.4692 / Windows 10.0.0
Referer: https://www.gbif.org/ar/species/2928783
Window size: width 1368 - height 801
[API log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2022-01-21T18:49:25.764Z',to:'2022-01-21T18:55:25.764Z'))&_a=(columns:!(_source),filters:!(),index:'3390a910-fcda-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
[Site log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2022-01-21T18:49:25.764Z',to:'2022-01-21T18:55:25.764Z'))&_a=(columns:!(_source),filters:!(),index:'5c73f360-fce3-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
System health at time of feedback: OPERATIONAL

**Comment by ManonGros** [comment]
All the observations concerned are from the Pl@ntNet automatically identified occurrences dataset: https://www.gbif.org/occurrence/search?country=SA&taxon_key=2928783&geometry=POLYGON((45.43945%2023.55469,47.84546%2023.19214,50.00977%2024.30176,43.98926%2027.99316,41.26465%2027.1582,45.43945%2023.55469))

https://www.gbif.org/dataset/14d5676a-2c54-4f94-9023-1e8dcd822aa0
https://www.gbif.org/publisher/da86174a-a605-43a4-a5e8-53d484152cd3

Edit: I notified Pl@ntNet.

**Comment by MortenHofft** [comment]
It is a bit odd to have machine identified observations based on images, and then not have the images nor a reference to where they can be seen on plantnet. E.g. https://www.gbif.org/occurrence/2647563725 
It is very hard for anyone to verify the occurrence. I would think the dataset was much more valuable if they included a link to the record on their own site. And possibly shared the image as well

**Comment by ManonGros** [comment]
Yes it isn't the first time this is mentioned: https://github.com/gbif/portal-feedback/issues/3124
